benchmarks/nemo/nemolite2d/manual_versions/psykal_sycl/Makefile

```diff
@@ -0,0 +1,63 @@
+# Makefile for the manual SYCL version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
+# environment variables. e.g. to use gfortran:
+# 
+# export F90=gfortran
```

Comment:
Probably these settings need to be updated (e.g. drop the OpenMP)?

Reply:
Removed OpenMP and AR